### PR TITLE
Fixed PHP8 ArgumentCountError with Mage_ConfigurableSwatches enabled.

### PR DESCRIPTION
**Pull request by S0FTWEX**
Fixed PHP8 ArgumentCountError with Mage_ConfigurableSwatches enabled.

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/core/Mage/Core/etc/config.xml [hunk 1]
```diff
@@ -485,9 +485,7 @@
                     <protected>
                         <app>/app/*/*</app>
                         <dev>/dev/*/*</dev>
-                        <downloader>/downloader/*/*</downloader>
```
Comment (colinmollenhour) [inline app/code/core/Mage/Core/etc/config.xml:488]: Is this safe to remove?

Comment (kkrieger85) [reply app/code/core/Mage/Core/etc/config.xml:488]: Yes. There is no "downloader" folder

**Review by kkrieger85 — APPROVED** [review]
Thx for your  PR

**Comment by S0FTWEX** [comment]
Duplicate.

**Pull request closed without merging**